### PR TITLE
wallet social profiles in dashboard

### PR DESCRIPTION
**Pull request by jnsdls**
before:
image

after:
image

---

## PR-Codex overview
This PR adds new components, updates URLs, and refactors imports across multiple files.

### Detailed summary
- Added `AppFooter` component and `TWAutoConnect` component
- Updated URLs for social API domain
- Removed `Spinner` import from multiple files
- Refactored import paths for better organization
- Added new components like `WalletAddress`, `MobileSidebar`, and `TabLinks`
- Updated package versions for `@radix-ui/react-avatar` and `@radix-ui/react-hover-card`
- Added new styling for `upsell` button class

> The following files were skipped due to too many changes: `apps/dashboard/src/components/wallets/ConnectWalletMiniPlayground/MiniPlayground.tsx`, `apps/dashboard/src/contract-ui/tabs/analytics/page.tsx`, `apps/dashboard/src/components/contract-components/contract-deploy-form/split-fieldset.tsx`, `apps/dashboard/src/components/contract-components/contract-deploy-form/custom-contract.tsx`, `apps/dashboard/src/contract-ui/tabs/manage/components/InstalledModulesTable.tsx`, `apps/dashboard/src/components/contract-components/contract-table-v2/index.tsx`, `apps/dashboard/src/components/pay/PayAnalytics/components/PaymentHistory.tsx`, `apps/dashboard/src/contract-ui/tabs/overview/components/MarketplaceDetails.tsx`, `apps/dashboard/src/contract-ui/tabs/nfts/components/table.tsx`, `apps/dashboard/src/pages/[chain_id]/[...paths].tsx`, `apps/dashboard/src/components/embedded-wallets/Users/index.tsx`, `apps/dashboard/src/components/engine/overview/engine-overview.tsx`, `apps/dashboard/src/components/engine/permissions/access-tokens-table.tsx`, `apps/dashboard/src/contract-ui/tabs/shared-components/marketplace-table.tsx`, `apps/dashboard/src/pages/dashboard/connect/analytics.tsx`, `apps/dashboard/src/app/(dashboard)/styleguide/page.tsx`, `apps/dashboard/src/contract-ui/tabs/account-permissions/components/account-signer.tsx`, `apps/dashboard/src/components/engine/engine-instances-table.tsx`, `apps/dashboard/src/components/engine/overview/backend-wallets-table.tsx`, `apps/dashboard/src/components/engine/overview/transactions-table.tsx`, `apps/dashboard/src/components/pay/PayAnalytics/components/PayCustomersTable.tsx`, `apps/dashboard/src/@/components/ui/hover-card.tsx`, `apps/dashboard/src/@/components/ui/avatar.tsx`, `apps/dashboard/src/@/components/blocks/wallet-address.tsx`, `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by jnsdls** [comment]
* **#4502** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4502?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jnsdls and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by gregfromstl — APPROVED** [review]

File: apps/dashboard/src/@/components/blocks/wallet-address.tsx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+          )}
+          <Button
+            asChild
+            variant="upsell"
+            className="text-sm flex flex-row gap-2 items-center"
+            size="sm"
+          >
+            <a
+              target="_blank"
+              href="https://blog.thirdweb.com/changelog/introducing-the-social-sdk?ref=dashboard-social-wallet"
```
Comment (gregfromstl) [inline apps/dashboard/src/@/components/blocks/wallet-address.tsx:150]: Linking to the feature changelog is a great idea

File: apps/dashboard/src/@/components/blocks/wallet-address.tsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  type SocialProfile,
+  useSocialProfiles,
+} from "thirdweb/react";
+import { resolveScheme } from "thirdweb/storage";
+import { setOverrides } from "../../../lib/vercel-utils";
+import { thirdwebClient } from "../../constants/client";
+import { Badge } from "../ui/badge";
+import { Button } from "../ui/button";
+
+setOverrides();
```
Comment (MananTank) [inline apps/dashboard/src/@/components/blocks/wallet-address.tsx:23]: looks like a left over?

File: apps/dashboard/src/app/layout.tsx [hunk 3]
```diff
@@ -7,9 +7,12 @@ import PlausibleProvider from "next-plausible";
 import dynamic from "next/dynamic";
 import { Inter } from "next/font/google";
 import NextTopLoader from "nextjs-toploader";
+import { setOverrides } from "../lib/vercel-utils";
 import { PostHogProvider } from "./components/root-providers";
 import { AppRouterProviders } from "./providers";
 
+setOverrides();
```
Comment (MananTank) [inline apps/dashboard/src/app/layout.tsx:14]: looks like a left over?

File: apps/dashboard/src/@/components/blocks/wallet-address.tsx [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+export default function WalletAddress(props: {
```
Comment (MananTank) [inline apps/dashboard/src/@/components/blocks/wallet-address.tsx:25]: Can we use named exports here - so its the same name everywhere?

**Review by MananTank — APPROVED** [review]
We can probably reduce the hover card delay a little

**Comment by MananTank** [comment]
Added to add translate-x to align with the table heading

**Comment by jnsdls** [comment]
### Merge activity

* **Sep 10, 2:49 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/thirdweb-dev/js/4502) disabled "merge when ready" on this PR due to: a merge conflict with the target branch; resolve the conflict and try again..

**Pull request merged**